README.md

```diff
@@ -1,6 +1,7 @@
 # A minimal logging API for Go
 
 [![Go Reference](https://pkg.go.dev/badge/github.com/go-logr/logr.svg)](https://pkg.go.dev/github.com/go-logr/logr)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/go-logr/logr/badge)](https://api.securityscorecards.dev/projects/github.com/go-logr/logr)
```

Comment:
This is exactly the snipped from https://github.com/ossf/scorecard#scorecard-badges. However, it links to a URL which just returns some JSON. Wouldn't it be better to use a link which shows some human-friendly rendering?

@pnacht: you used https://deps.dev/go/github.com%2Fgo-logr%2Flogr for that. Is that a link that we can use here?